ppr-ui/src/components/common/CautionBox.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div class="caution-box ma-0 pa-4 fs-14" :class="{ 'alert-box': setAlert }">
     <slot name="prependSLot" />
-    <p><b>{{ setImportantWord }}:</b> <span v-html="setMsg" /></p>
+    <v-icon v-if="setAlert" color="error" class="alert-icon mr-2">mdi-alert</v-icon>
```

Comment:
This might affect `Dashboard.vue` line 12-32 when the ProductStatus is rejected. Might want to do some regression testing on that.